app/views/MessagesView/index.js

```diff
@@ -225,6 +232,10 @@ class MessagesView extends React.Component {
 		}
 	}
 
+	setFileLoading = (fileLoading) => {
```

Comment:
`setFileLoading = fileLoading => this.setState({ fileLoading });` ?

Reply:
if I do it this way I get eslint:  error    `Unexpected parentheses around single function argument having a body with no curly braces  arrow-parens`

Reply:
Only if you put:
`setFileLoading = (fileLoading) => this.setState({ fileLoading });`
You need to remove parentheses on params :)